tests/plugins/imagebase/features/manual/upload.html

```diff
@@ -1,14 +1,16 @@
+<script>
+	if ( isUnsupportedEnvironment() ) {
```

Comment:
It should not be inserted into a separate `script` element. There's a script element just few lines below.

Reply:
There was some troubles with this test on old IE, but I've made full if-else statement. I hope now it will work properly.